Source/Charts/Components/YAxis.swift

```diff
@@ -150,6 +150,24 @@ open class YAxis: AxisBase
         var min = _customAxisMin ? _axisMinimum : dataMin
         var max = _customAxisMax ? _axisMaximum : dataMax
         
+        // Make sure max is greater than min
+        // Discussion: https://github.com/danielgindi/Charts/pull/3650#discussion_r221409991
+        if min > max
+        {
+            if _customAxisMax && _customAxisMin
```

Comment:
I think this would be easier to read as a switch statement:
```
switch (customAxisMax, customAxisMin) {
case (true, true):
    (min, max) = (max, min)

case (true, false):
    min = max * 0.5

case (false, true):
    max = min * 1.5

case (false, false):
    break
}
```

Reply:
thanks!